googlecast/src/main/java/com/kaltura/playkit/plugins/googlecast/caf/basic/PlaybackParams.java

```diff
@@ -0,0 +1,198 @@
+            playbackParams.put(SOURCES, new JSONObject(sourcesJson));
+        } catch (JSONException e ) {
+            e.printStackTrace();
+        }
+        return playbackParams;
+    }
+
+    public void setDashSource(String id, String url, String licenseUrl) {
+        Dash dashSource =  new Dash();
+        dashSource.id = id;
+        dashSource.url = url;
+        dashSource.mimetype = APPLICATION_DASH_XML;
+
+        DrmData drmData = new DrmData();
+        drmData.licenseUrl = licenseUrl;
+        drmData.scheme = COM_WIDEVINE_ALPHA;
+        dashSource.drmData = Collections.singletonList(drmData);
+        this.dash = Collections.singletonList(dashSource);
+    }
+
+    public void setHlsSource(String id, String url) {
+        Hls hlsSource =  new Hls();
+        hlsSource.id = id;
+        hlsSource.url = url;
+        hlsSource.mimetype = APPLICATION_X_MPEG_URL;
+        this.hls = Collections.singletonList(hlsSource);
+    }
+
+    public void setProgressivehSource(String id, String url) {
+        Progressive progressivSource =  new Progressive();
+        progressivSource.id = id;
+        progressivSource.url = url;
+        progressivSource.mimetype = VIDEO_MP_4;
+
+        this.progressive = Collections.singletonList(progressivSource);
+    }
+
+    public String getPoster() {
+        return poster;
+    }
+
+    public void setPoster(String poster) {
+        this.poster = poster;
+    }
+
+    public List<Hls> getHls() {
+        return hls;
+    }
+
+    public void setHls(List<Hls> hls) {
+        this.hls = hls;
+    }
+
+    public List<Dash> getDash() {
+        return dash;
+    }
+
+    public void setDash(List<Dash> dash) {
+        this.dash = dash;
+    }
+
+    public List<Progressive> getProgressive() {
+        return progressive;
+    }
+
+    public void setProgressive(List<Progressive> progressive) {
+        this.progressive = progressive;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public Integer getDuration() {
+        return duration;
+    }
+
+    public void setDuration(Integer duration) {
+        this.duration = duration;
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    public void setType(String type) {
+        this.type = type;
+    }
+
+    public Boolean getDvr() {
+        return dvr;
+    }
+
+    public void setDvr(Boolean dvr) {
+        this.dvr = dvr;
+    }
+
+    public Object getVr() {
+        return vr;
+    }
+
+    public void setVr(Object vr) {
+        this.vr = vr;
+    }
+
+    public Metadata getMetadata() {
+        return metadata;
+    }
+
+    public void setMetadata(Metadata metadata) {
+        this.metadata = metadata;
+    }
+
+    public List<Caption> getCaptions() {
+        return captions;
+    }
+
+    public void setCaptions(List<Caption> captions) {
+        this.captions = captions;
+    }
+}
```

Comment:
Do we really all of these getters and setters?

Reply:
no removing it